text/0000-components.md

```diff
@@ -0,0 +1,261 @@
+systems integrators. Initially, only the Rust programming language is
+supported, though others may be proposed in future RFCs.
+
+In particular, the SDK includes APIs and implementations to help write smart
+contracts, including:
+
+- Reading product information from state
+- Reading Pike identity information from state
+- Reading schema information from state
+- Pike permission verifier
+- Schema compliance verify
+
+Like the other components, the SDK will grow over time as new features are
+supported by Grid. When adding new features or making changes to the SDK,
+backwards compatibility should be considered. The SDK’s versioning should
+conform to semver (which is true of all components, but of particular
+importance for the SDK).
+Smart Contracts
+
+Several features of Hyperledger Grid use smart contracts as part of their
+implementation. The smart contracts are deployed in Sabre, which is part of
+Hyperledger Sawtooth. Sabre executes smart contracts which have been compiled
+to WebAssembly. Most of the smart contracts which are formally part of
+Hyperledger Grid will be written in the Rust programming language. However,
+future RFCs may extend this to additional programming languages supported by
+future versions of Sabre.
+
+Some initial smart contracts supported by Grid are:
+
+- Grid Product
+- Grid Pike
+- Grid Track and Trace
+- Grid Schemas
+
+Smart contracts added to Grid will likely always be accompanied by an RFC which
+documents the problem they solve and technical details needed to write and
+maintain the smart contract.
+
+## Documentation
+
+Hyperledger Grid’s documentation plays a critical role in describing Grid to
+potential users and providing necessary guide and reference material for
+application developers, systems integrators, and systems administrators.
+
+The team feels strongly that “documentation is source code” and the approach to
+maintaining it matches the processes we use for all other development. As such,
+we have chosen formats for documentation which are friendly to source control
+under git while providing the best output possible. The documentation is
+written in: a) Rust doc comments for SDK API documentation; b) sphinx-doc for
+the bulk of non-API documentation; c) Markdown friendly for viewing directly in
+github for specific other documents.
+
+The sphinx-doc documentation will include an introduction to Grid, user guides,
+developer guides, system administration guides, CLI reference, smart contract
+specifications, and more.
+
+As new additions to Grid are added the documentation should be updated as well.
+
+# Reference-level explanation
+The initial organization of the grid repository shall be:
+
+`grid/bin/` - Convenience scripts for developers. This directory is never
+packaged or installed.
+
+`grid/cli/` - The Grid CLI command source code.
+
+`grid/contracts/<contract>/` - Smart contract source code, with each smart
+contract having a separate directory “<contract>”.
```

Comment:
`<contract>` has weird formatting. try using ` instead of ”

Reply:
Markdown (and Sphinx) don't generally let you mix fixed-width and italic fonts in the same string.  The docs use `{` curly braces `}` for this reason (not < angle brackets > ). 

I'd also suggest using contract-name instead of contract.  But that's a nitpick.